pkg/chart/chart.go

```diff
@@ -197,12 +197,23 @@ func (t *Testing) processCharts(action func(chart string, valuesFiles []string)
 		return nil, errors.Wrap(err, "Error initializing Helm")
 	}
 
+	repoArgs := map[string][]string{}
+
+	for _, repo := range t.config.HelmRepoExtraArgs {
+		repoSlice := strings.SplitN(repo, "=", 2)
+		name := repoSlice[0]
+		repoExtraArgs := strings.Fields(repoSlice[1])
+		repoArgs[name] = repoExtraArgs
+	}
+
 	for _, repo := range t.config.ChartRepos {
 		repoSlice := strings.SplitN(repo, "=", 2)
 		name := repoSlice[0]
 		url := repoSlice[1]
-		if err := t.helm.AddRepo(name, url); err != nil {
-			return nil, errors.Wrapf(err, "Error adding repo: %s=%s", name, url)
+
+		repoExtraArgs := repoArgs[name]
+		if err := t.helm.AddRepo(name, url, repoExtraArgs); err != nil {
+			return nil, errors.Wrapf(err, "Error adding repo: %s=%s with args %s", name, url, repoExtraArgs)
```

Comment:
Don't include the args in the error message. We don't want to log sensitive data.